src/programs/ps-matrix/ps-matrix

```diff
@@ -139,81 +133,40 @@ def readHandData(data, handcol, valuecol, norm):
     return handvals
 
 
-def usage():
-    print('usage: %s [options]' % sys.argv[0])
-    print('       ')
-    print('       print data values as a preflop holdem matrix')
-    print('       ')
-    print('       options:')
-    print(
-        '       --hand-column,h <N>          column number to find hand, default=0'
-    )
-    print(
-        '       --value-column,v <N>         column number to find value, default=1'
-    )
-    print('       --normalize,n <pf|data|none> normalize flag, default=none')
-    print(
-        '       --file <filename>            file to find data, default=stdin')
-    print('       ')
-    print('       The columns need to be space separated')
-    print('       ')
-    print('       ')
-    print('       examples:')
-    print('           cat data.txt | ./pfmatrix.py')
-    print('           ./pfmatrix.py --file data.txt')
-    print(
-        '           ./pfmatrix.py --normalize --hand-column 2 --data-column 8 --file data.txt'
-    )
-
-
 def main(argv=None):
-    if argv is None:
-        argv = sys.argv
-    try:
-        (opts, args) = getopt.getopt(argv[1:], '?h:v:n:f:t:', [
-            'help',
-            'hand-column=',
-            'value-column=',
-            'normalize=',
-            'file=',
-            'threshold=',
-        ])
-    except getopt.GetoptError, e:
-        print(e)
-        usage()
-        sys.exit(2)
-    if len(sys.argv) == 2 and not '-n' in sys.argv[1]:
-        usage()
-        sys.exit(2)
+    parser = ap.ArgumentParser(
+        description=__doc__,
+        epilog='The columns need to be space separated.',
+        formatter_class=ap.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        '-p',  # '-h' conflicts w/ --help, so '-p' is for pocket.
```

Comment:
For the purposes of maintaining compatibility with other scripts we should disable the -h default.

See: https://stackoverflow.com/a/14951061/132401